indexing-hadoop/pom.xml

```diff
@@ -67,6 +67,10 @@
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-client</artifactId>
     </dependency>
+      <dependency>
+          <groupId>org.apache.hadoop</groupId>
+          <artifactId>hadoop-aws</artifactId>
```

Comment:
hadoop-aws shouldn't be necessary after  #1151 unless you're really running it on aws with native s3 fs.